# HG changeset patch
# User Pantxo Diribarne <pantxo.diribarne@gmail.com>
# Date 1503496566 -7200
#      Wed Aug 23 15:56:06 2017 +0200
# Node ID b208e1a7dd023d2f5faac0ce3640ec8c8ddc278a
# Parent  174b6afa7e7f32ce7f8259f948ecec3af6468b0c
imshow.m: fix colormap related demos (bug #51825)

diff --git a/scripts/image/imshow.m b/scripts/image/imshow.m
--- a/scripts/image/imshow.m
+++ b/scripts/image/imshow.m
@@ -219,17 +219,17 @@ endfunction
 
 %!demo
 %! clf;
 %! imshow ("default.img");
 
 %!demo
 %! clf;
 %! imshow ("default.img");
-%! colormap (autumn (64));
+%! colormap (gca, autumn (64));
 
 %!demo
 %! clf;
 %! [I, M] = imread ("default.img");
 %! imshow (I, M);
 
 %!demo
 %! clf;
@@ -240,17 +240,17 @@ endfunction
 %!demo
 %! clf;
 %! imshow (rand (100, 100));
 %! title ({"imshow with random 100x100 matrix", "black and white"});
 
 %!demo
 %! clf;
 %! imshow (rand (100, 100));
-%! colormap (jet (64));
+%! colormap (gca, jet (64));
 %! title ({"imshow with random 100x100 matrix", "colormap() makes color image"});
 
 %!demo
 %! clf;
 %! imshow (rand (100, 100, 3));
 %! title ({"imshow with random 100x100x3 matrix", "RGB color"});
 
 %!demo
